framework/auth/views.py

```diff
@@ -245,13 +245,8 @@ def auth_login(auth, **kwargs):
         next_url = request.args.get('redirect_url')
         must_login_warning = False
 
-    if next_url:
```

Comment:
Since you've removed this check, what are the possible values for next_url? If next_url is None, then your validate_next_url() function will raise an attribute error that you won't be checking for.